javascript/javascript1/week1/houseprice.js

```diff
@@ -0,0 +1,43 @@
+const widthOfHouse = [8, 5];
+const heightOfHouse = [10, 8];
+const depthOfHouse = [10, 11];
+const gardenSizeInM2 = [100, 70];
+let volumeInMeters = widthOfHouse[1] * heightOfHouse[1] * depthOfHouse[1];
+
+const housePricePeter = volumeInMeters * 2.5 * 1000 + gardenSizeInM2[0] * 300;
+const actualPricePeter = 2500000;
+
+if (housePricePeter > actualPricePeter) {
+  console.log(
+    "The cost of Peter's house is " +
+      housePricePeter +
+      "." +
+      " Peter is paying too much!"
+  );
+} else {
+  console.log(
+    "The cost of Peter's house is " +
+      housePricePeter +
+      "." +
+      " Peter is paying too little!"
+  );
+}
+
+const housePriceJulia = volumeInMeters * 2.5 * 1000 + gardenSizeInM2[1] * 300;
+const actualPriceJulia = 1000000;
+
+if (housePriceJulia > actualPriceJulia) {
+  console.log(
+    "The cost of Julia's house is " +
+      housePriceJulia +
+      "." +
+      " Julia is paying too much!"
+  );
+} else {
+  console.log(
+    "The cost of Julia's house is " +
+      housePriceJulia +
+      "." +
+      " Julia is paying too little!"
+  );
+}
```

Comment:
!OBS! Something is a little off with this solution 💡

You have a good approach and you are almost there 😊

**Suggestions** 
- Try to calculate the actualPrice for both Julia and Peter without the code and see if you get the same values as the result your code provides. If not, why could that be?
- Look at the variable volumeInMeters. Are you using the same values for both Julia and Peter? How could you make it so, that they have separate values for the variable volumeInMeters based on their own houses' width, height and depth? 

I hope this makes sense. Don't hesitate to reach out to me on Slack if you want to talk it through. 😊
My Slack name is @Marie Louise Meyer 🙋🏼

Reply:
I think I will spend some time revisiting this task as it did get a litttttle confusing there - great points made, I'll rework it